projects/xmlunit/project.yaml

```diff
@@ -0,0 +1,5 @@
+homepage: "https://github.com/xmlunit/xmlunit"
+language: jvm
+primary_contact: "irali@google.com"
+main_repo: "https://github.com/xmlunit/xmlunit.git"
+file_github_issue: true
```

Comment:
Please set this to false. Otherwise we may start spamming maintainers with potentially bad crash reports.